api-event-files/index.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+$url = 'https://plausible.io/api/event';
+
+$headers = [];
+
+$allowed_headers =
+
+	[
+		'Accept',
+		'Accept-Language',
+		'Connection',
+		'Content-Length',
+		'Content-Type',
+		'Cookie',
+		'Origin',
+		'Referer',
+		'Sec-Ch-Ua',
+		'Sec-Ch-Ua-Mobile',
+		'Sec-Ch-Ua-Platform',
+		'Sec-Fetch-Dest',
+		'Sec-Fetch-Mode',
+		'Sec-Fetch-Site',
+		'User-Agent',
+		'X-Forwarded-Host',
+		'X-Forwarded-Port',
+		'X-Forwarded-Proto',
+		'X-Forwarded-Scheme',
+		'X-Original-Host',
+		'X-Real-Ip',
+		'X-Request-Id',
+		'X-Scheme',
+	];
+
+foreach ( getallheaders() as $key => $value ) {
+	if ( in_array( $key, $allowed_headers ) ) {
+		$headers[ $key ] = sprintf( '%s: %s', $key, $value );
+	}
+}
+
+
+if ( ! empty( $_SERVER['HTTP_CLIENT_IP'] ) ) {
+	$ip = $_SERVER['HTTP_CLIENT_IP'];
+} elseif ( ! empty( $_SERVER['HTTP_X_FORWARDED_FOR'] ) ) {
+	$ip = $_SERVER['HTTP_X_FORWARDED_FOR'];
+} else {
+	$ip = $_SERVER['REMOTE_ADDR'];
```

Comment:
What's the purpose of the `$ip` variable? Don't see it being used below in this file.